PYME/LMVis/Extras/pointSetGeneration.py

```diff
@@ -134,14 +134,23 @@ def OnConfigure(self, event):
         self.edit_traits()
 
     def OnGenPoints(self, event):
-        self.xp, self.yp, self.zp = self.source.getPoints()
+        import wx
+        try:
```

Comment:
these checks are needed as otherwise one is left in an unworkable state and needs to exit the app as configuration cannot be fixed; therefore there are similar changes in all other places